File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    s = s.lower()
    numP = 0;
    numY = 0;
    for i in range(len(s)):
        if(s[i] == 'p'):
            numP += 1
        if(s[i] == 'y'):
            numY += 1
    if(numP == 0 and numY == 0):
        return True
    return numP == numY

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    s = s.toLowerCase();
    var answer = true;
    var numY = 0;
    var numP = 0;
    for(let i = 0; i < s.length; i++){
        if(s[i] == 'y'){ numY++; }
        else if(s[i] == 'p'){ numP++; }
    }

    return numP == numY;
}